data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,804 @@
+	thumbnail "outfit/orchid missile launcher"
+	"mass" 4.5
+	"outfit space" -18
+	"weapon capacity" -18
+	"gun ports" -1
+	"orchid capacity" 12
+	weapon
+		sprite "projectile/thunderhead fragment"
+			"frame rate" 1
+			"no repeat"
+		sound "finisher"
+		ammo `"Orchid" Nuclear Missile`
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"fire effect" "speckle fire"
+		"inaccuracy" 2.4
+		"velocity" 9
+		"lifetime" 30
+		"reload" 60
+		"firing energy" 6
+		"firing heat" 3
+		"radar tracking" 0.6
+		"infrared tracking" 0.9
+		"missile strength" 36
+		"submunition" "Orchid Active"
+		stream
+	description "Unlike humanity, the Avgi do not have a strong taboo against the use of nuclear weapons in space. To them, nuclear weapons are simply another tool, albeit one with a particularly high energy density, to be used when the occasion calls for it. Nuclear weapons have been the Avgi's trump card against the Aberrant, the one weapon they have capable of reliably breaking their advanced shields and atomizing their armor."
+	description "A standard modular launcher such as this one is designed to be easily installed on light to medium warships."
+
+
+outfit "Orchid Launch Array"
+	category "Secondary Weapons"
+	licenses
+		"Avgi Atomics"
+	cost 1370000
+	thumbnail "outfit/unknown"
+	"mass" 4.8
+	"outfit space" -48
+	"weapon capacity" -48
+	"turret mounts" -1
+	"orchid capacity" 48
+	weapon
+		sprite "projectile/thunderhead fragment"
+			"frame rate" 1
+			"no repeat"
+		sound "finisher"
+		ammo `"Orchid" Nuclear Missile`
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"inaccuracy" 45
+		"velocity" 3
+		"lifetime" 30
+		"reload" 30
+		"burst reload" 12
+		"burst count" 3
+		"turret turn" 9
+		"firing energy" 3
+		"radar tracking" 0.6
+		"infrared tracking" 0.9
+		"missile strength" 36
+		"submunition" "Orchid Active"
+		"stream"
+
+
+outfit "Orchid Active"
+	weapon
+		sprite "projectile/thunderhead fragment"
+			"frame rate" 2.4
+		"hit effect" "thermonuclear flash"
+		"hit effect" "prompt radiation" 9
+		"hit effect" "fission fragments" 240
+		"live effect" "nettle exhaust" 1800
+		"die effect" "missile death"
+		"lifetime" 1800
+		"acceleration" 0.6
+		"drag" .03
+		"turn" 9
+		"homing" 4
+		"radar tracking" 0.9
+		"infrared tracking" 0.6
+		"trigger radius" 60
+		"blast radius" 180
+		"shield damage" 9000
+		"hull damage" 6000
+		"hit force" 3600
+		"missile strength" 42
+
+
+outfit `"Snapdragon" Nuclear Shaped Charge`
+	category "Ammunition"
+	licenses
+		"Avgi Atomics"
+	cost 660000
+	thumbnail "outfit/casaba howitzer"
+	"mass" 1
+	"snapdragon capacity" -1
+	description "Casaba Howitzer (campaign unlock, not needed for initial PR)"
+
+
+outfit "Snapdragon Magazine"
+	category "Ammunition"
+	licenses
+		"Avgi Atomics"
+	cost 1440000
+	thumbnail "outfit/unknown"
+	"mass" 3
+	"outfit space" -18
+	"snapdragon capacity" 12
+	ammo `"Snapdragon" Nuclear Shaped Charge`
+	description "Snapdragon storage"
+
+
+outfit "Snapdragon Launcher"
```

Comment:
This is much more reasonable than the Orchid. It's a larger launcher with a longer reload, but more importantly the mechanics of the missile are both interesting and open to counterplay. The damage is higher on paper, but between the spread of the projectiles, the effective dropoff caused by the random velocity and random lifetime, and the fact that that displayed damage also includes submunitions that are sent flying backwards, it's considerably lower in practice than the displayed numbers. On top of this, it doesn't always hit - often, only a glancing blow is scored, and I strongly suspect that as a player taking fire from these there'd be some trick to reliably avoid them once you'd worked out how they worked.
These, I think, are a far more reasonable weapon than the Orchid, and a much better idea of what these kinds of weapons should look like from the gameplay side.
Don't get me wrong though, they're still extremely effective and really impressive when used, they're just not all-powerful like the Orchid.